.laminas-ci.json

```diff
@@ -0,0 +1,18 @@
+{
+    "additional_checks": [
+        {
+            "name": "QA Checks",
+            "job": {
+                "php": "*",
+                "command": "composer check",
+                "dependencies": ["lowest", "latest"]
+            }
+        },
+        {
+            "name": "Lint ruleset.xml using phpcs.xsd schema",
+            "job": {
+                "command": "xmllint --noout --schema vendor/squizlabs/php_codesniffer/phpcs.xsd src/LaminasCodingStandard/ruleset.xml"

```

Comment:
xmllint is not installed by default.

Maybe we should move this into it's own action file and then we can install it with this:

```
- name: Install xmllint
  run: sudo apt-get install libxml2-utils
```